src/v/kafka/server/replicated_partition.cc

```diff
@@ -282,22 +282,26 @@ raft::replicate_stages replicated_partition::replicate(
 std::optional<model::offset> replicated_partition::get_leader_epoch_last_offset(
   kafka::leader_epoch epoch) const {
     const model::term_id term(epoch);
-    const auto first_local_term = _partition->get_term(
-      _partition->start_offset());
-    // found in local state
+    const auto first_local_offset = _partition->start_offset();
```

Comment:
we should post the link to the code of the java upstream here

Reply:
Done